### ISSUE TITLE
$container-background-colo undefined

### ISSUE THREAD

**Issue by billyboozer** [foundation/foundation-emails#223]
# Looks like there's a missing variable in the _grid.scss file ($container-background-color).

```
Message:
    node_modules/foundation-emails/scss/grid/_grid.scss
Error: Undefined variable: "$container-background-color".
        on line 31 of node_modules/foundation-emails/scss/grid/_grid.scss
>>     background: $container-background-color;
   ----------------^
```

I created a pull request https://github.com/zurb/foundation-emails/pull/222

**Comment by gakimball** [comment]
Fixed via #222

**Comment by billyboozer** [comment]
Awesome. Thanks! :+1:
